winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# Content Types in WinUI AvatarView
+
+The `AvatarView` control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.
```

Comment:
The default content type displays the `Avatar1` character image when initializing the control without the initials, custom image, or group source.